src/strings.cpp

```diff
@@ -665,6 +665,9 @@ struct UnitConversion {
 struct Units {
 	UnitConversion c; ///< Conversion
 	StringID s;       ///< String for the unit
+	/// Number of decimal places embedded in the value. For example, 1 if
+	/// the value is in tenths, and 3 if the value is in thousandths.
+	unsigned int places;
```

Comment:
Better off putting this on one line (OTTD doesn't have any line length limits, and needs `///<` to be a valid doxygen comment anyway)
```suggestion
	uint places; ///< Number of decimal places embedded in the value. For example, 1 if the value is in tenths, and 3 if the value is in thousandths.
```

Reply:
`places` is only used as a boolean in the code, it should also be a boolean here

Reply:
> Better off putting this on one line (OTTD doesn't have any line length limits, and needs `///<` to be a valid doxygen comment anyway)

The doxygen syntax is fine, see the "third alternative" in https://www.doxygen.nl/manual/docblocks.html#cppblock, but I changed it anyway for consistency with the adjacent lines.

> `places` is only used as a boolean in the code, it should also be a boolean here

No, it's used as an integer, like the comment says. The value of `places` ultimately becomes the last argument in the call to `FormatCommaNumber` here: https://github.com/jostephd/OpenTTD/blob/2f7d15a6705cca9a77e17368ea9cfb2a04eee20d/src/strings.cpp#L1053-L1057 So if you set `places` to 2, the displayed figure will have two decimal places, and so on.